diff --git a/lib/git.js b/lib/git.js
index 57d58f5..3be4b88 100644
--- a/lib/git.js
+++ b/lib/git.js
@@ -139,11 +139,7 @@ Git.prototype.exec = function () {
  * Remove all unversioned files.
  * @return {Promise} A promise.
  */
-Git.prototype.rm = function (files) {
-  if (!Array.isArray(files)) {
-    files = [files];
-  }
-  return this.exec('rm', '--ignore-unmatch', '-r', '-f', '--', ...files);
+Git.prototype.rm = function () {
+  return this.exec('rm', '--ignore-unmatch', '-r', '-f', '--', '.');
 };
