diff --git a/lib/git.js b/lib/git.js
index d4c5724..05378f5 100644
--- a/lib/git.js
+++ b/lib/git.js
@@ -139,11 +139,8 @@ Git.prototype.checkout = function (remote, branch) {
  * @param {string | Array<string>} files Files argument.
  * @return {Promise} A promise.
  */
-Git.prototype.rm = function (files) {
-  if (!Array.isArray(files)) {
-    files = [files];
-  }
-  return this.exec('rm', '--ignore-unmatch', '-r', '-f', '--', ...files);
+Git.prototype.rm = function () {
+  return this.exec('rm', '--ignore-unmatch', '-r', '-f', '--', '.');
 };
 
 /**
